README.md

```diff
@@ -50,3 +50,18 @@ sudo pip install -r requirements.txt
 sudo pip install -r dev_requirements.txt
 cp greenpithumb/wiring_config.ini.example greenpithumb/wiring_config.ini
 ```
+
+## QuickStart Instructions
+
+Run greenpithumb/greenpithumb.py and supply command-line flags for poll_interval, photo_interval, config_file, sleep_window, image_path, and db_file. Example below.
```

Comment:
Suggest something like "After running the installation steps above on your Pi, run GreenPiThumb with a command like the following"